rpc/core/blocks.go

```diff
@@ -81,6 +82,14 @@ func Block(ctx *rpctypes.Context, heightPtr *int64) (*ctypes.ResultBlock, error)
 	return &ctypes.ResultBlock{BlockID: blockMeta.BlockID, Block: block}, nil
 }
 
+// BlockByHash gets block of given hash.
+// More: https://tendermint.com/rpc/#/Info/block_by_hash
+func BlockByHash(ctx *rpctypes.Context, hash bytes.HexBytes) (*ctypes.ResultBlock, error) {
```

Comment:
```suggestion
func BlockByHash(ctx *rpctypes.Context, hash []byte) (*ctypes.ResultBlock, error) {
```

What is the purpose of hash being `bytes.HexBytes`?

Reply:
Everywhere is bytes.HexBytes.

Reply:
@melekes Internally the type of bock hash is `bytes.HexBytes`. That's why I have used this.